### PR TITLE
Fix indentation/spacing in comments to render correctly in godoc

### PR DESCRIPTION
**Pull request by songxiao-wang87**
What type of PR is this?
/kind cleanup
/kind documentation

What this PR does / why we need it:
Fixes incorrect indentation/line breaks inserted by go1.19 gofmt

Which issue(s) this PR fixes:
none

Special notes for your reviewer:
none

Does this PR introduce a user-facing change?
none

```release-note

NONE

```

### CODE REVIEW

**Review by yangjunmyfm192085 — COMMENTED** [review]
/retest
/lgtm

**Comment by aojea** [comment]
The failure is legit

> {Failed  === RUN   TestServiceAccountTokenCreate/a_token_is_valid_against_the_HTTP-provided_service_account_issuer_metadata
>     svcaccttoken_test.go:746: get token
>     svcaccttoken_test.go:765: get discovery doc
>     svcaccttoken_test.go:796: raw discovery doc response:
>         ---{"issuer":"https://foo.bar.example.com","jwks_uri":"https://openid/v1/jwks","response_types_supported":["id_token"],"subject_types_supported":["public"],"id_token_signing_alg_values_supported":["ES256"]}
> 


can you split independent changes on independent commits? 
it simplifies backports and is easier to maintain the repo

**Comment by songxiao-wang87** [comment]
The test code is correct, this part of the modification has been rolled back.@aojea

File: test/e2e/storage/framework/testdriver.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ const (
 	CapSnapshotDataSource Capability = "snapshotDataSource" // support populate data from snapshot
 	CapPVCDataSource      Capability = "pvcDataSource"      // support populate data from pvc
 
-	// multiple pods on a node can use the same volume concurrently;
+	// CapMultiPODs on a node can use the same volume concurrently;
```
Comment (sftim) [inline test/e2e/storage/framework/testdriver.go:160]: This change is _not_ a fix to indentation or spacing.

Comment (songxiao-wang87) [reply test/e2e/storage/framework/testdriver.go:160]: comment on exported type Repo should be of the form "XXX ..." (with optional leading article). So I changed it to match the spec！

File: pkg/apis/certificates/types.go [hunk 2]
```diff
@@ -192,7 +192,7 @@ type CertificateSigningRequestList struct {
 	Items []CertificateSigningRequest
 }
 
-// KeyUsages specifies valid usage contexts for keys.
+// KeyUsage specifies valid usage contexts for keys.
```
[the remark this answers is not in the document]
Comment (songxiao-wang87) [reply pkg/apis/certificates/types.go:195]: comment on exported type Repo should be of the form "XXX ..." (with optional leading article). So I changed it to match the spec！

**Pull request closed without merging**